roles/testnode/tasks/setup-ubuntu.yml

```diff
@@ -12,6 +12,6 @@
   lineinfile:
     dest: /etc/lvm/lvm.conf
     regexp: "# types ="
-    line: 'types = [ "rbd", 1024 ]'
+    line: 'types = [ "rbd", 16 ]'
```

Comment:
For xfstests purposes, the value isn't actually important, so I'm not worried about some nodes having 1024 and other nodes having 16.